go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/supabase-community/realtime-go
 
 go 1.17
```

Comment:
we can upgrade this version.

Reply:
which version should we go up to? the latest one?

Reply:
`go 1.22` is better choise maybe

Reply:
Done!